### ISSUE TITLE
Acrobat reader mangles display of fonts with em > 1000

### ISSUE THREAD

**Issue by derde** [fontforge/fontforge#5185]
When xelatex renders OTF fonts with em size > 1000, the resulting PDF file looks great on GPL PDF viewers.  On Adobe Acrobat, however, the fonts are scaled by around 1.7 times (see picture).

The warning in FontForge says "Non-standard Em-Sizse" and "The convention is that PostScript fonts should have an Em-Size of 1000. But this font has a size of 1786. This is not an error, but you might ..."  It would help to have a warning here about Adobe's own software not understanding non-standard Em-Size.  This might cause subtle problems for em-sizes like 1024.

Adobe's html/javascript on-line site https://acrobat.adobe.com/ does show this error.  Interestingly, their thumbnails seem to be generated in a way that does not show the error.

![image](https://user-images.githubusercontent.com/2578422/214282064-c241efb4-4525-423c-bec5-dbbb6fe52763.png)

**Comment by lemzwerg** [comment]
... probably a bug in the Adobe software – you should contact them.

**Comment by derde** [comment]
Agreed, it's a bug in Acrobat - but not giving the appropriate warning of the foreign bug is something of a bug in FontForge.  I've tried to report it to Adobe - but I'm not their client, so I'm not sure they will care.

**Comment by Panogen** [comment]
I'm not sure we should hold FontForge responsible for bugs that have nothing to do with FontForge itself.

**Comment by derde** [comment]
FontForge presents a warning, but the warning is presented as "information".  That's a minor bug, but a bug.  However, I see FontForge can load the font from the PDF, but FontForge **also** assumes that the font size is 1000.  This is a similar bug to the bug in Adobe Acrobat.  The font elements overflow the (assumed) boundaries:
![image](https://user-images.githubusercontent.com/2578422/214531060-26ce9b4f-ba40-4cbc-b8c1-acbcfcf68b8c.png)

**Comment by ctrlcctrlv** [comment]
Are you just saying that the window shouldn't be called Information? I can reopen this and mark it "Documentation".

**Comment by derde** [comment]
It should be "WARNING", as not even fontforge can handle fonts where this value is not 1000.
